### PR TITLE
Thomas/wip

### PR DESCRIPTION
**Pull request by fel-thomas**
### Documentation enhancement

f5d70a53e128273b3a51634c3381723fa5ecf51d Remove unused pages (my bad, I should have removed them long time ago)

2dd873b4c7acf521e2d3f274eb86e9bbdc7eb733 The Creator and Ackno section were different between readme and the doc

2dd873b4c7acf521e2d3f274eb86e9bbdc7eb733 2dd873b4c7acf521e2d3f274eb86e9bbdc7eb733 2dd873b4c7acf521e2d3f274eb86e9bbdc7eb733 2dd873b4c7acf521e2d3f274eb86e9bbdc7eb733 11333331de3e42bcba86ef03cf9f620e8fcaa338 aed5cb93a8a90241c7aa2561227a1278118b0eb5 aed5cb93a8a90241c7aa2561227a1278118b0eb5 are the api or typo in the documentation :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -234,11 +235,17 @@ To learn more about Explainable AI in general, see:
 ## 🙏 Acknowledgments
 
 <img align="right" src="https://www.deel.ai/wp-content/uploads/2021/05/logo-DEEL.png" width="25%">
-This project received funding from the French ”Investing for the Future – PIA3” program within the Artificial and Natural Intelligence Toulouse Institute (ANITI). The authors gratefully acknowledge the support of the <a href="https://www.deel.ai/"> DEEL </a> project namely from [Justin Plakoo](https://github.com/justinplakoo) and [Lucas Hervier](https://github.com/lucashervier).
+This project received funding from the French ”Investing for the Future – PIA3” program within the Artificial and Natural Intelligence Toulouse Institute (ANITI). The authors gratefully acknowledge the support of the <a href="https://www.deel.ai/"> DEEL </a> project.
```
Comment (lucashervier) [inline README.md:238]: Yes my bad ;)

File: docs/api/insertion.md [hunk 2]
```diff
@@ -20,10 +20,11 @@ from xplique.attributions import Saliency
 
 # load images, labels and model
 # ...
-explainer = Saliency(model)
+explainer = Saliency(model, )
```
Comment (lucashervier) [inline docs/api/insertion.md:23]: Why the coma ?

Comment (fel-thomas) [reply docs/api/insertion.md:23]: oups =O thx ! I'll fix that now

File: docs/css/custom.css [hunk 3]
```diff
@@ -6,6 +6,20 @@
     --synthax-red: #ec407a;
 }
 
+[data-md-color-scheme="slate"] {
```
Comment (lucashervier) [inline docs/css/custom.css:9]: Nice job ;)

File: xplique/attributions/deconvnet.py [hunk 4]
```diff
@@ -56,7 +56,7 @@ def explain(self,
         Returns
         -------
         explanations
-            Guided Backpropagation maps.
+            Deconv maps.
```
Comment (lucashervier) [inline xplique/attributions/deconvnet.py:59]: Nice catch!

**Review by lucashervier — COMMENTED** [review]
LGTM :) I think there is only a forgot comma, I pin it in my comments!

**Review by lucashervier — APPROVED** [review]

**Comment by fel-thomas** [comment]
Done for the comma ;)

**Pull request merged**